about.html

```diff
@@ -0,0 +1,80 @@
+                    <li id="tw-li"><img id="tw" src="img/login/login_tw.png" alt="twitter icon"></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </nav>
+        </div>
+        <div class="navbar1">
+            <div id="logo"><a href="index.html">ABDOU SHOP</a></div>
+            <div id="open-menu"><button id="open-menu-button" type="submit"><img src="img/header/open-menu.png" alt="open menu"></button></div>
+            <nav class="navbar">
+                <div id="close-menu"><button id="close-menu-button" type="submit"><img src="img/header/close-menu.png" alt="close menu"></button></div>
+                <ul class="navbar-links">
+                    <li><a class="navbar-link" id="first-link" href="index.html">HOME</a></li>
+                    <li><a class="navbar-link" id="second-link" href="#services">SERVICES</a></li>
+                    <li><a class="navbar-link" href="about.html">ABOUT</a></li>
+                    <li><a href="#"></a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <div id="about">
+        <h5>SHOP WHAT YOU LOVE FASTER AND EASIER.</h5>
+        <p class="h5-supportingText">Our store started on 08 May 2019 , in Algeria, Our goal is to let the customer enjoy differents types of clothes , in any categories , with good quality and cheap price than other stores</p>
+        <div class="questions">
+            <p>Please contact us per Email for any further questions about our store , or to be an affiliate marketer with us !</p>
+            <a href="mailto:belkaid.abdulah@gmail.com">belkaid.abdulah@gmail.com</a>
+        </div>
+    </div>
+    
+    <div class="maps">
+        <h1>Our Stores : </h1>
+        <div class="line"></div>
+        <div class="maps-container">
+            <div class="map-1">
+                <p id="first-store"> The first store :</p>
+                <div class="map1"><iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3198.6315577280675!2d3.003317214600373!3d36.70739068061956!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x128fafea0eab462f%3A0xb80f309646e46877!2sH%26M%20Draria!5e0!3m2!1sen!2sdz!4v1652566922315!5m2!1sen!2sdz" width="600" height="450" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe></div>
+            </div>
+            <div class="map-2">
+                <p id="second-store"> The second store :</p>
+                <div class="map2"><iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3197.48728510794!2d2.9472578187599137!3d36.73487312438183!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x128fafa39749d2e7%3A0xce2888f1f8563b01!2sUK%20Fashion%20Store!5e0!3m2!1sen!2sdz!4v1652567026773!5m2!1sen!2sdz" width="600" height="450" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe></div>
```

Comment:
- [ ] (OPTIONAL) Also to improve your UI design, you could add the background overlay color on your maps 👍 
- Your design:
![Annotation 2022-05-18 110635](https://user-images.githubusercontent.com/85113567/169014707-dfa349b4-0086-4ac4-8e09-fd48473b1352.jpg)


- Required design
![Annotation 2022-05-18 110524](https://user-images.githubusercontent.com/85113567/169014609-2f09ccf3-cab1-4dda-b381-8e449c302275.jpg)
: